go/web/skeleton/Makefile

```diff
@@ -41,14 +42,32 @@ p2p-build: lint service-build service-push ## Builds the service image and pushe
 .PHONY: p2p-functional
 p2p-functional: namespace = $(tenant_name)-functional
 p2p-functional: build-functional push-functional create-ns-functional deploy-functional ## Runs only functional helm tests
-	helm test $(app_name) -n $(namespace) --filter name=$(app_name)-functional-test
-	@echo Logs: $(call logs_url,"$(app_name)-functional-test",$(current_timestamp))
+	$(eval POD_NAME := $(app_name)-functional-test)
+	helm test $(app_name) -n $(namespace) --filter name=$(POD_NAME) --timeout $(helm_test_timeout_duration)
```

Comment:
Looks like running tests can be templated (create a unified makefile target) and each p2p-* target can just pass different arguments: $(POD_NAME).

But I don't insist

Reply:
yeap, good idea, maybe I'll merge this for now and do that when making changes next time to save time on testing as it is quite time consuming.